modules/atomic-widgets/widgets/atomic-image.php

```diff
@@ -13,28 +13,65 @@
 }
 
 class Atomic_Image extends Atomic_Widget_Base {
-	public function get_icon() {
-		return 'eicon-image';
+	public function get_name() {
+		return 'a-image';
 	}
 
 	public function get_title() {
 		return esc_html__( 'Atomic Image', 'elementor' );
 	}
 
-	public function get_name() {
-		return 'a-image';
+	public function get_icon() {
+		return 'eicon-image';
 	}
 
 	protected function render() {
 		$settings = $this->get_atomic_settings();
 
-		$image_url = $settings['image']['url'];
+		$attrs = array_filter( array_merge(
+			$settings['image'] ?? [],
+			[ 'class' => $settings['classes'] ?? '' ]
+		) );
+
+		Utils::print_wp_kses_extended(
+			sprintf(
+				'<img %1$s >',
+				Utils::render_html_attributes( $attrs )
+			),
+			[ 'image' ]
+		);
+	}
+
+	protected function define_atomic_controls(): array {
+		return [
+			Section::make()
+				->set_label( esc_html__( 'Content', 'elementor' ) )
+				->set_items( [
+					Attachment_Control::bind_to( 'image' )
+						->set_media_types( [ 'image' ] ),
+
+					Select_Control::bind_to( 'image_size' )
+						->set_label( esc_html__( 'Image Resolution', 'elementor' ) )
+						->set_options( $this->get_image_size_options() ),
+				]),
+		];
+	}
+
+	protected static function define_props_schema(): array {
+		return [
+			'classes' => Atomic_Prop::make(),
 
-		?> <img
-			src='<?php echo esc_url( $image_url ); ?>'
-			alt='Atomic Image'
-		/>
-		<?php
+			'image' => Atomic_Prop::make()
+				->default( [
+					'$$type' => 'image-url',
+					'value' => [
+						'url' => Utils::get_placeholder_image_src(),
+					],
+				] ),
+
+			'image_size' => Atomic_Prop::make()
+				->default( 'large' ),
```

Comment:
aren't you using this value in the render method?

Reply:
We should talk about it, for now image is not part of this PR